### ISSUE TITLE
Missing accessibility support

### ISSUE THREAD

**Issue by arun-sasidharan** [bokuweb/react-rnd#858]
### Overview of the problem

I'm using react-rnd **version** [x.x.x]   
      
My **browser** is:   
   
I am sure this issue is **not a duplicate**?
   
### Reproduced project

https://codesandbox.io/s/y3997qply9

### Description

We want our app to fully accessible to people with disabilities. It is not currently supported by react rnd. We expect the drag and drop to work for everyone.

1. User will grab the draggable using a space key.
2. User will move the draggable using kepad arrows.
3. User will drop the draggable using enter and release grab using space.

### Steps to Reproduce

1. First Step
4. Second Step
5. and so on...
